docs/getting-started/advanced-setting.asciidoc

```diff
@@ -71,9 +71,9 @@ collectors to ship data to {es}, the data must be mapped to ECS.
 [[update-threat-intel-indices]]
 == Update default Elastic Security threat intelligence indices
 
-The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat indicators. The setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
+The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} features use when querying for ingested threat indicators. The setting controls features that query threat intelligence indices, such as the Threat Intelligence view on the Overview page, indicator match rules, and the alert enrichment query. One or more threat intelligence indices can be defined; multiple indices must be separated by commas. By default, the `logs-ti*` index pattern is defined and used by {agent} integrations.
```

Comment:
```suggestion
The `securitySolution:defaultThreatIndex` advanced setting specifies threat intelligence indices that {elastic-sec} features query for ingested threat indicators. This setting affects features that query threat intelligence indices, such as the Threat Intelligence view on the Overview page, indicator match rules, and the alert enrichment query. You can specify one or more threat intelligence indices; multiple indices must be separated by commas. By default, only the `logs-ti*` index pattern is included. 
```
I suggested to remove "and used by {agent} integrations" because Elastic agent integrations aren't mentioned earlier in the paragraph, so I thought it was confusing to add it at the end. As a relatively naive user it left me wondering if agent integrations are in the category of features that query the threat intelligence indices for threat indicators, or if they were another category of thing that also somehow uses the same setting.

Reply:
I'm completely on board with all of the edits, but do want to keep the bit about Elastic agent integrations. In 7.x, the default threat intel index was populated with Filebeat data (as opposed to data from agent integrations) and named differently (`filebeat-*`) so I think it's necessary to include this distinction.